views/apartments/show.erb

```diff
@@ -1 +1,5 @@
-This is info about apartment #3.
+<!-- This is info about apartment #3. -->
+
+<h2><%= @apartment.address %></h2>
+
+<h4><%= @apartment.monthly_rent %></h4>
```

Comment:
I would expect to see more information about this Apartment on the show page.  If you did one as an example, then moved on - leave a comment here, for me and you. :)